manticore/ethereum.py

```diff
@@ -655,17 +661,49 @@ def make_symbolic_value(self):
         return ABI.SValue
 
     @staticmethod
-    def compile(source_code, contract_name=None):
+    def compile(source_code, contract_name=None, libraries=None):
         ''' Get initialization bytecode from a Solidity source code '''
-        name, source_code, bytecode, runtime, srcmap, srcmap_runtime, hashes, abi, warnings = ManticoreEVM._compile(source_code, contract_name)
+        name, source_code, bytecode, runtime, srcmap, srcmap_runtime, hashes, abi, warnings = ManticoreEVM._compile(source_code, contract_name, libraries)
         return bytecode
 
     @staticmethod
-    def _compile(source_code, contract_name):
+    def _link(bytecode, libraries=None):
+        has_dependencies = '_' in bytecode
```

Comment:
Might be good to explain it somewhere why `_` in bytecode means it has dependencies (or give link to some source?)
(is it that obvious?)

Reply:
A link to some doc should be good to have.
(bytecode should be hex.. I assume the _ will be in all the link placeholders)